README.md

```diff
@@ -133,7 +133,7 @@ The main purpose of this repository is to continue evolving open source. We want
 Facebook has adopted a Code of Conduct that we expect project participants to adhere to.
 Please read the [full text][code] so that you can understand what actions will and will not be tolerated.
 
-[code]: https://code.fb.com/codeofconduct/
+[code]: https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CODE_OF_CONDUCT.md
```

Comment:
@vj-codes  you should use the link as a simple version **(./CODE_OF_CONDUCT.md)**